dlt/destinations/impl/snowflake/configuration.py

```diff
@@ -60,6 +63,7 @@ class SnowflakeCredentials(ConnectionStringCredentials):
     authenticator: Optional[str] = None
     private_key: Optional[TSecretStrValue] = None
     private_key_passphrase: Optional[TSecretStrValue] = None
+    application: Optional[str] = None
```

Comment:
1. I think you can just set the default here
2. `application_id` looks like a better name? or we want to be compatible with snowflake conn param name? if the latter you can keep `application`

Reply:
I think it is best to be compatible with standard snowflake parameters because `application_id` will create more confusion for end users.